### PR TITLE
Drop type specs for Muon and RecoB

### PR DESCRIPTION
**Pull request by NiharSaha**
#### PR description:
Drop type specs in DQMOffline/Muon and DQMOffline/RecoB python configuration files.

Central DQM migration campaign for drop type spces following
https://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1
and update the safer syntax for existing parameters like: (1) drop type specifications where the original parameter exists. (2) using "clone" instead of "deepcopy" (3) move all parameters inside the clone

#### PR validation:

Tested in CMSSW_12_3_X via runTheMatrix.py -l limited -i all --ibeos

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

File: DQMOffline/RecoB/python/dqmCollector_cff.py [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
 bTagCollectorSequenceDATA = cms.Sequence(bTagCollectorDATA)
 
 #pf MC collector
 bTagCollectorMC = bTagHarvestMC.clone(
-    flavPlots = cms.string("allbcl"), #harvest all, b, c, dusg and ni histos 
-    ptRanges = cms.vdouble(0.0),
-    etaRanges = cms.vdouble(0.0),
+    flavPlots = "allbcl", #harvest all, b, c, dusg and ni histos 
+    ptRanges = (0.0,),
+    etaRanges = (0.0,),
```
Comment (perrotta) [inline DQMOffline/RecoB/python/dqmCollector_cff.py:14]: Is this syntax correct? Maybe yes... But why not sticking on the usual way of listing vectors in python with squared parentheses, e.g.:
```suggestion
    ptRanges = [0.0],
    etaRanges = [0.0],
```

Comment (NiharSaha) [reply DQMOffline/RecoB/python/dqmCollector_cff.py:14]: @perrotta Okay! I am considering changing parentheses as you mentioned. Thank you.

**Comment by perrotta** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**